app/src/shared/web3/web3Provider.js

```diff
@@ -25,6 +25,7 @@ export class StreamrWeb3 extends Web3 {
     constructor(provider: any, options: StreamrWeb3Options = {}) {
         super(provider)
         this.isLegacy = options && !!options.isLegacy
+        this.transactionConfirmationBlocks = getConfig().transactionConfirmationBlocks
```

Comment:
What is this used for?

Reply:
Local Ganache-blockhain will confirm transactions immediately so we need to inform web3 about that. By default web3 requires 24 confirmations which will not happen locally. We need to set this value to get the transaction receipt.

Reply:
Got it 👍 maybe let's add that as a comment as well.